XRTK-Core/Assets/XRTK/Inspectors/PropertyDrawers/TypeReferencePropertyDrawer.cs

```diff
@@ -347,22 +345,22 @@ private static void DisplayDropDown(Rect position, List<Type> types, Type select
             menu.AddItem(new GUIContent("(None)"), selectedType == null, OnSelectedTypeName, null);
             menu.AddSeparator(string.Empty);
 
-            for (int i = 0; i < types.Count; ++i)
+            foreach (var type in types)
             {
-                string menuLabel = FormatGroupedTypeName(types[i], grouping);
+                var menuLabel = FormatGroupedTypeName(type, grouping);
```

Comment:
Why the sudden change of explicit types to using var, we already know the format and type for each of these.

Reply:
I think it's pretty obvious that it's a string from the context.